src/Test/Perf/util/test_util.csx

```diff
@@ -111,11 +116,21 @@ string ReleaseVbcPath()
     return Path.Combine(BinReleaseDirectory(), "vbc.exe");
 }
 
+string GetCPCDirectoryPath()
+{
+    return Environment.ExpandEnvironmentVariables(@"%SYSTEMDRIVE%\CPC");
+}
+
+string GetCPCSourceBinaryLocation()
+{
+    return $@"\\mlangfs1\public\basoundr\CpcBinaries";
```

Comment:
Well, they have no plans to have this released as nuget. What they are willing to do is give up binaries which will have public APIs into which we can call into. Again, these libraries will not be public. What this means is, instead of having to call the `CPC.exe /Start`, we might call `CPC.start()` as an API.